src/features/collapsible-sidebar/CollapsibleSidebar.js

```diff
@@ -90,8 +93,16 @@ class CollapsibleSidebar extends React.Component<Props> {
             className,
         );
 
+        const htmlAttributes = isHidden
+            ? { ...this.props.htmlAttributes, 'aria-hidden': 'true' }
+            : this.props.htmlAttributes;
```

Comment:
`this.props.htmlAttributes` could stay destructured and the new `const htmlAttributes` be renamed.

Could also create another object specifically aria attributes (e.g. https://github.com/box/box-ui-elements/blob/master/src/components/text-area/TextArea.js#L55)